src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Security/SecurityHeaderNames.cs

```diff
@@ -0,0 +1,13 @@
+namespace OrchardCore.Security
```

Comment:
Why is this file in Infrastructure? Everything should be contained in the new module

Reply:
It could, I added it there because it could be used in any other part / module in the system